__tests__/app.test.js

```diff
@@ -116,15 +116,63 @@ describe.only("GET /api/articles/:article_id", () => {
         });
     });
 
-    it('should return a 400 Bad Request error when endpoint provided an id that is the wrong data type', () => {
-        return request(app)
-      .get("/api/articles/hello")
-      .expect(400)
-      .then((res) => {
-        const body = res.body;
-        expect(body).toEqual({ message: "Bad Request" });
-      });
+    it("should return a 400 Bad Request error when endpoint provided an id that is the wrong data type", () => {
+      return request(app)
+        .get("/api/articles/hello")
+        .expect(400)
+        .then((res) => {
+          const body = res.body;
+          expect(body).toEqual({ message: "Bad Request" });
+        });
     });
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  it("should return an array of comments with the corresponding article_id with properties comment_id, votes, created_at author, body", () => {
+    const articleId = 3;
+    return request(app)
+      .get(`/api/articles/${articleId}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual([
```

Comment:
Consider iterating over the results and using a more flexible jest matcher here to check what each result contains. (avoid toEqual).